src/modules/image-upload/image-upload.js

```diff
@@ -72,31 +72,54 @@ function getSignature(policy) {
 export function getResponse(policyReq) {
 	const keyPrefix = getKeyPrefix(policyReq.auth.user, policyReq.uploadType, policyReq.textId),
 		policy = getPolicy(keyPrefix),
-		signature = getSignature(policy);
+		signature = getSignature(policy),
+		baseURL = 'https://up.bel.ng/',
+		fileName = policyReq.fileName.replace(/\s+/g, ' ');
+	let key = keyPrefix,
+		thumbnail = baseURL + keyPrefix.replace(/^uploaded/, 'generated'),
+		url;
+
+	switch (policyReq.uploadType) {
+	case 'avatar':
+		key += 'original.' + fileName.split('.').pop();
+		url = baseURL + key;
+		thumbnail += '256x256.jpg';
+		break;
+	case 'content':
+		key += '1/' + fileName;
+		url = baseURL + keyPrefix + '1/' + encodeURIComponent(fileName);
+		thumbnail += '1/480x960.jpg';
+		break;
+	}
 
 	policyReq.response = {
-		acl: config.s3.acl,
-		policy,
-		keyPrefix,
-		bucket: config.s3.bucket,
-		'x-amz-algorithm': config.s3.algorithm,
-		'x-amz-credential': getCredential(),
-		'x-amz-date': getDate(true),
-		'x-amz-signature': signature,
+		key,
+		url,
+		thumbnail,
+		policy: {
+			acl: config.s3.acl,
+			policy,
+			'x-amz-algorithm': config.s3.algorithm,
+			'x-amz-credential': getCredential(),
+			'x-amz-date': getDate(true),
```

Comment:
Can you change this function to `getDate({ long: true })`? `getDate` is confusing.